include/faabric/transport/MpiMessageEndpoint.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
```

Comment:
Currently, the implementation of these is so simple, that they are alright with just being abstract methods. However, I thought they belonged in `transport` rather than `scheduler`.